content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,78 @@
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/docs/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
+- **Added** alpha support for the OpenCensus trace driver.
+- **Improved** reporting for external services by removing requirements to add a service entry.
+- **Improved** the mesh dashboard to provide monitoring of Istio's configuration state.
+- **Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+- **Removed** deprecated `Adapter` and `Template` custom resource definitions (CRDs).
+- **Deprecated** Mixer adapters. Please begin the transition of all extensions to the new extension model. Legacy Mixer integration support continues only for Istio 1.3 and 1.4.
+- **Deprecated** the HTTP API spec used to produce API attributes. We will remove support for producing API attributes in Istio 1.4.
+
+## Policy
+
+- **Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration management
+
+- **Fixed** Galley to stop too many gRPC pings from closing connections.
+- **Improved** Galley to avoid control plane upgrade failures.
+
+## `istioctl`
+
+- **Added** [`istioctl experimental manifest`](/docs/reference/commands/istioctl/#istioctl-experimental-manifest) to manage the new experimental install manifests.
+- **Added** [`istioctl experimental profile`](/docs/reference/commands/istioctl/#istioctl-experimental-profile) to manage the new experimental install profiles.
+- **Added** [`istioctl experimental metrics`](/docs/reference/commands/istioctl/#istioctl-experimental-metrics)
+- **Added** [`istioctl experimental describe pod`](/docs/reference/commands/istioctl/#istioctl-experimental-describe-pod) to describe an Istio pod's configuration.
+- **Added** [`istioctl experimental add-to-mesh`](/docs/reference/commands/istioctl/#istioctl-experimental-add-to-mesh) to add Kubernetes services or virtual machines to an existing Istio service mesh.
+- **Added** [`istioctl experimental remove-from-mesh`](/docs/reference/commands/istioctl/#istioctl-experimental-remove-from-mesh) to remove Kubernetes services or virtual machines from an existing Istio service mesh.
+- **Promoted** the [`istioctl experimental metrics`](/docs/reference/commands/istioctl/#istioctl-experimental-metrics) command to `istioctl metrics`.
```

Comment:
These three promoted sections look clunky in the release notes.  Any suggestions welcome - Im about out of steam.